index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+      <link rel="stylesheet" href="style.css">
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Poetry</title>
+  </head>
+   <body style="background-color:lightblue;">
+  <body>
+    
+    <p>
+      <h1> 
+  <strong>ON PAPER</strong>
+    </h1> 
+    </p>
+    
+<p>
+ <h2>
+   By Jacqueline Woodson
+  </h2> 
+    </p>
```

Comment:
Same note here about removing the `p` element
```suggestion
 <h2>
   By Jacqueline Woodson
  </h2> 
```